### PR TITLE
Fixes broken ember-app JShint tests

### PR DESCRIPTION
**Pull request by petey-rock**
Fixes broken ember-app JShint tests

### CODE REVIEW

**Comment by discorick** [comment]
Awesome! Lets wait on merging this until #280 makes its way in, theres guarantee'd to be some more jshint fixes needed, and likely a merge conflict.

**Comment by dahlbyk** [comment]
@petey-rock now that #280 has shipped can you merge `master` in here and make sure we're still happy?

**Comment by discorick** [comment]
@dahlbyk would a rebase not be better?

**Comment by dahlbyk** [comment]
@petey-rock can you fix the two warnings introduced with #280?

File: ember-app/tests/unit/services/board-syncing-test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ moduleFor('service:board-syncing', {
 
 test('syncs the boards issues successfuly', (assert)=>{
   var issues = ['issue1', 'issue2'];
-  var success = $.ajax().then(()=>{return issues});
+  var success = $.ajax().then(()=>{return issues;});
```
Comment (dahlbyk) [inline ember-app/tests/unit/services/board-syncing-test.js]: FYI, `then(() => issues);` is also valid.

Comment (discorick) [inline ember-app/tests/unit/services/board-syncing-test.js]: Yep, For inline functions I still lean towards explicit `return` declarations... for readability sake more than anything (this isn't ruby after all :D )

File: ember-app/app/vendor/jquery.color.contrast.js [hunk 2]
```diff
@@ -4,5 +4,5 @@
     var r = this._rgba[0], g = this._rgba[1], b = this._rgba[2];
     return (((r*299)+(g*587)+(b*144))/1000) >= 131.5 ? "#333" : "white";
   };
-})(jQuery);
+})();
```
Comment (dahlbyk) [inline ember-app/app/vendor/jquery.color.contrast.js]: This means `$` will be undefined above.

Comment (discorick) [inline ember-app/app/vendor/jquery.color.contrast.js]: Reverting the change and replacing `$` with `Ember.$` should satisfy jhint unless its not able to work with iffys

Comment (dahlbyk) [inline ember-app/app/vendor/jquery.color.contrast.js]: You mean replacing the `jQuery` IIFE argument with `Ember.$`?

Comment (discorick) [inline ember-app/app/vendor/jquery.color.contrast.js]: Yes, @petey-rock what @dahlbyk said...

**Comment by dahlbyk** [comment]
Last three commits here look like `master` was rebased on to this branch, rather than the other way around?

**Pull request closed without merging**